frontend/public/components/utils/_value-from-pair.scss

```diff
@@ -1,6 +1,9 @@
 .value-from-pair {
-  float: left;
-  width: 50%;
+  padding-top: 5px;
+  @media(min-width: $screen-md-min) {
+    float: left;
```

Comment:
Not new to this PR, but this classname is misleading as each `.value-from-pair` contains a single `.dropdown`.  Would love to see both `.dropdown`s in a single `.value-from-pair` then CSS like:

```
// this is untested
.value-from-pair {
  display: flex;
  flex-direction: column;
  @media(min-width: $screen-md-min) {
    flex-direction: row;
  }
  .dropdown {
    .width: 50%;
  }
}
```

Reply:
So, the dropdown creates this grouping of <div>s. I pass in the class name and it creates the <div class=classname><div class=dropdown></div></div>. Unless you want a new double dropdown common class created, I don't think what you are asking is possible...but lets chat and make sure.